bin/ruby_version_test

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-rubies=("ruby-2.4.4" "ruby-2.5.0" "ruby-2.5.1")
+rubies=("ruby-2.4.4" "ruby-2.5.0" "ruby-2.5.1" "ruby-2.5.3" "ruby-2.5.3" "ruby-2.6.1")
```

Comment:
2.5.3 is there twice... 2.5.2?

Reply:
oops, cheers.

The script version is only useful for running locally, and perhaps, if a fork were setup it would be runnable without any CI.